NebulaHost/PacketProcessors/Logistics/ILSgStationPoolSyncRequestProcessor.cs

```diff
@@ -0,0 +1,128 @@
+﻿using NebulaModel.Attributes;
+using NebulaModel.Networking;
+using NebulaModel.Packets.Logistics;
+using NebulaModel.Packets.Processors;
+using NebulaModel.DataStructures;
+using System.Collections.Generic;
+
+namespace NebulaHost.PacketProcessors.Logistics
+{
+    [RegisterPacketProcessor]
+    public class ILSgStationPoolSyncRequestProcessor : IPacketProcessor<ILSRequestgStationPoolSync>
+    {
+        private PlayerManager playerManager;
+        public ILSgStationPoolSyncRequestProcessor()
+        {
+            playerManager = MultiplayerHostSession.Instance.PlayerManager;
+        }
+        public void ProcessPacket(ILSRequestgStationPoolSync packet, NebulaConnection conn)
+        {
+            Player player = playerManager.GetPlayer(conn);
+            if(player == null)
+            {
+                player = playerManager.GetSyncingPlayer(conn);
+            }
+            if (player != null)
+            {
+                List<int> stationGId = new List<int>();
```

Comment:
This is still ulsing lists and could be using arrays though.

Reply:
ups